.github/libs/GithubUtils.js

```diff
@@ -118,23 +118,23 @@ class GithubUtils {
      * @returns {Array<Object>} - [{url: String, number: Number, isVerified: Boolean}]
      */
     static getStagingDeployCashPRList(issue) {
-        let PRListSection = issue.body.match(/pull requests:\*\*\r?\n((?:.*\r?\n)+)\r?\n/) || [];
+        let PRListSection = issue.body.match(/pull requests:\*\*(?:\r?\n)*((?:.*\r?\n(?:\s+-\s.*\r?\n)+\r?\n)+)/) || [];
         if (PRListSection.length !== 2) {
             // No PRs, return an empty array
             console.log('Hmmm...The open StagingDeployCash does not list any pull requests, continuing...');
             return [];
         }
         PRListSection = PRListSection[1];
         const unverifiedPRs = _.map(
-            [...PRListSection.matchAll(new RegExp(`- \\[ ] (${PULL_REQUEST_REGEX.source})`, 'g'))],
+            [...PRListSection.matchAll(new RegExp(`(${PULL_REQUEST_REGEX.source})[\r\n]\\s+- \\[ \\] QA`, 'g'))],
```

Comment:
This is a little funky because you're matching for a carriage return OR a newline, followed by 1 or more whitespace characters (which can be a space, tab, carriage return, newline, etc...

Maybe it'll be less breakable and work just as well if we just do:

```suggestion
            [...PRListSection.matchAll(new RegExp(`(${PULL_REQUEST_REGEX.source})\\s+- \\[ \\] QA`, 'g'))],
```